angular-locale_en-cn.js

```diff
@@ -80,10 +80,10 @@ angular.module("ngLocale", [], ["$provide", function($provide) {
                 5,
                 6
             ],
-            "fullDate": "y\u5e74M\u6708d\u65e5EEEE",
-            "longDate": "y\u5e74M\u6708d\u65e5",
-            "medium": "y\u5e74M\u6708d\u65e5 ah:mm:ss",
-            "mediumDate": "y\u5e74M\u6708d\u65e5",
+            "fullDate": "y, MMMM d, EEEE",
+            "longDate": "y, MMMM d ",
+            "medium": "y, MMM d ah:mm:ss",
+            "mediumDate": "y, MMM d ",
```

Comment:
Please remove commas and extra M's. For example, fullDate should look like this: 
"y M d EEEE" also remove all trailing spaces because that might show up oddly.